go/pkg/reader/reader.go

```diff
@@ -13,6 +13,8 @@ import (
 	"github.com/mostynb/zstdpool-syncpool"
 )
 
+var errNotInitialized = errors.New("Not yet initialized")
```

Comment:
+1 to golint complaining about the capitalization here

Reply:
That would technically be an API change since the capitalization was present in the original code, so I'd rather not change it as part of this bug fix.

Reply:
Sounds reasonable. This complaint comes from golint-ci, could you add a `//nolint:golint` inline comment per [this article](https://golangci-lint.run/usage/false-positives/#nolint-directive)?

Reply:
Done.